### ISSUE TITLE
metacritic.com banner/sidebar ads

### ISSUE THREAD

**Issue by prech** [uBlockOrigin/uAssets#6260]
```
https://www.metacritic.com/browse/tv/score/metascore/90day/filtered?sort=desc
https://www.metacritic.com/tv/ramy/season-1
```

### Describe the issue

Header, sidebar and footer ads

### Screenshot(s)

https://i.imgur.com/eXYxF6f.jpg
https://i.imgur.com/ryANrca.jpg

### Versions

- Browser/version: Chrome 76.0.3809.132
- uBlock Origin version: 1.22.0

### Settings

- I tried a few Selector elements, but the ads are dynamic with new names each load

**Comment by dumbusernameidk** [comment]
Instart Logic ads site needs to be added to UBOE.

**Comment by llacb47** [comment]
@dumbusernameidk can you test these filters without using that scriplet? They seem to get rid of ads and g00 crap for me.

```
@@||metacritic.com^$generichide
||metacritic.com^*/ad?i10c.getabdurl=*&ad_channel=$media,redirect=noop.txt
```

**Comment by dumbusernameidk** [comment]
Yes that works for me.

**Comment by prech** [comment]
Would it be possible to integrate these working filters into a future filter set?

Much appreciated!

**Comment by okiehsch** [comment]
"okiehsch closed this in 1832499"
I closed the issue with the linked commit.
**llacb47** filters have been added to `uBO-fiters`.

**Comment by prech** [comment]
> I closed the issue with the linked commit.
> **llacb47** filters have been added to `uBO-filters`.

Ah, wonderful 

Thank you
